eng/Versions.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>3.3.1</VersionPrefix>
+    <NetAnalyzersVersionPrefix>5.0.0</NetAnalyzersVersionPrefix>
     <AnalyzerUtilitiesVersionPrefix>$(VersionPrefix)</AnalyzerUtilitiesVersionPrefix>
     <PreReleaseVersionLabel>beta1</PreReleaseVersionLabel>
```

Comment:
Can we have a `PreReleaseVersionLabel` for the `NetAnalyzers` package that is different than the other packages in this repo? For example I would like to lable the current package in master `rc1` as that is the .NET 5 release it is going into

Reply:
Yep, let me push a follow-up commit for it.